draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -103,49 +97,69 @@ This specification defines the format of the Client Attestation that a Client In
 # Terminology
 
 Client Attestation JWT:
-:  A JSON Web Token (JWT) generated by the client backend which is bound to a key managed by a client instance which can then be used by the instance for client authentication.
+:  A JSON Web Token (JWT) generated by the client backend which is bound to a key managed by a Client Instance which can then be used by the instance for client authentication.
 
 Client Attestation Proof of Possession (PoP) JWT:
-:  A Proof of Possession generated by the client instance using the key that the Client Attestation JWT is bound to.
+:  A Proof of Possession generated by the Client Instance using the key that the Client Attestation JWT is bound to.
+
+Client Instance:
+: A deployed instance of a piece of client software.
 
 Client Instance Key:
-:  A cryptographic, asymmetric key generated by the client instance and proven to the client backend. The public key is contained in the Client Attestation JWT and is used to sign the Client Attestation Proof of Possession.
+:  A cryptographic, asymmetric key generated by the Client Instance and proven to the client backend. The public key is contained in the Client Attestation JWT and is used to sign the Client Attestation Proof of Possession.
+
+# Client Attestation
 
-# Client Authentication
+This draft introduces the concept of client attestations to the OAuth 2 protocol, which consists of two JWT's, a Client Attestation and Client Attestation PoP, which are conveyed via HTTP headers in an HTTP request sent by a client to an Authorization Server or Resource Server. The purpose of these headers is to authenticate or attest to the Client Instance.
```

Comment:
The concept of assertions was introduced in OAuth in RFC 7521 and RFC 7523 introduced use of JWTs as assertions to authenticate clients. This draft adds key binding on top of JWT-based client authentication.

Reply:
As above I think this is a more general terminology change we will tackle seperately, we have created an issue.